src/ubloxGPS.cpp

```diff
@@ -449,6 +460,11 @@ int ubloxGPS::setOn(lib_config_t &config)
     return SYSTEM_ERROR_NONE;
 }
 
+void ubloxGPS::getAttitude(ubx_attitude_t *o)
+{
+    memcpy(o, &this->attitude, sizeof(ubx_attitude_t));
```

Comment:
Two requests:
1. Check `o` is nonzero first.
2. Use `sizeof(*o)` so that if the type of `o` changes, the statement is still valid.